cadquery/occ_impl/exporters/assembly.py

```diff
@@ -99,6 +105,171 @@ def exportAssembly(
     return status == IFSelect_ReturnStatus.IFSelect_RetDone
 
 
+def exportStepMeta(
+    assy: AssemblyProtocol,
+    path: str,
+    write_pcurves: bool = True,
+    precision_mode: int = 0,
+) -> bool:
+    """
+    Export an assembly to a STEP file with faces tagged with names and colors. This is done as a
+    separate method from the main STEP export because this is not compatible with the fused mode
+    and also flattens the hierarchy of the STEP.
+
+    Layers are used because some software does not understand the ADVANCED_FACE entity and needs
+    names attached to layers instead.
+
+    :param assy: assembly
+    :param path: Path and filename for writing
+    :param names: A dictionary of that specifies faces that need to have string names attached.
+    :param colors: A dictionary of that specifies faces that need to have RGB colors attached.
+    :param layers: A dictionary of that specifies faces that need to have string layer names attached.
```

Comment:
Can remove unused params from docstring.

Reply:
Thanks, I removed those from the docstring and it looks like @adam-urbanczyk opened #1814 to capture and discuss the idea of adding modes.